code/modules/hydroponics/grown/root.dm

```diff
@@ -62,6 +62,48 @@
 	juice_typepath = /datum/reagent/consumable/parsnipjuice
 	wine_power = 35
 
+/obj/item/food/grown/parsnip/attackby(obj/item/I, mob/user, params)
+	if(I.get_sharpness())
+		var/parsnip_blade
```

Comment:
this should be typed as an item

Reply:
what do you meeeean

Reply:
similar to the clothing thing, parsnip_blade is never gonna be a wall, or a number. it's always gonne be some sort of item so it should be typed like that

Reply:
Added a comments to clarify. Is that acceptable?